<launch>

<node name="manipulator_controller" pkg="m4e_mani_impedance_control" type="manipulator_controller" output="screen">
    <rosparam command="load" file="$(find m4e_mani_impedance_control)/config/params_sim.yaml"/>
</node>

</launch>